cmd/cmd.go

```diff
@@ -525,6 +526,50 @@ func RunServer(_ *cobra.Command, _ []string) error {
 	return server.Serve(ln)
 }
 
+func startMacApp(client *api.Client) error {
+	link, err := os.Readlink("/usr/local/bin/ollama")
```

Comment:
This isn't a good check if the user runs ollama outside of the mac app. It should probably call instead `os.Readlink(os.Executable())`